cloudevents/bindings/nats-protocol-binding.md

```diff
@@ -1,4 +1,4 @@
-# NATS Protocol Binding for CloudEvents - Version 1.0.3-wip
+# NATS Protocol Binding for CloudEvents - Version 1.0.4-wip
```

Comment:
I think the name remains the same since we're still technically in the 1.0.3 cycle.

Reply:
Reverted.